packages/social-previews/package.json

```diff
@@ -0,0 +1,50 @@
+{
+	"name": "@automattic/social-previews",
+	"version": "1.0.0",
+	"description": "A suite of components to generate previews for a post for both social and search engines",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"sideEffects": false,
+	"keywords": [
+        "wordpress",
+        "social",
+        "seo",
+        "search"
+	],
+	"author": "Automattic Inc.",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/social-previews"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"files": [
+		"dist",
+		"src"
+	],
+	"scripts": {
+		"clean": "check-npm-client && npx rimraf dist",
+		"prepublish": "check-npm-client && yarn run clean",
+		"prepare": "check-npm-client && transpile && copy-assets"
+	},
+	"dependencies": {
+		"i18n-calypso": "^5.0.0",
```

Comment:
Minor thing: these should be alphabetically sorted. Not a big deal, because the next time you add/remove a dependency, `yarn` will sort the list anyway.

Reply:
Thanks. If it doesn't cause an issue then I"ll leave it then. Waiting for CI is bad enough as it is! 😄